ompi/op/op.h

```diff
@@ -59,20 +59,20 @@ BEGIN_C_DECLS
  * repeated code, but it's better this way (and this typedef will
  * never change, so there's not much of a maintenance worry).
  */
-typedef void (ompi_op_c_handler_fn_t)(void *, void *, int *,
+typedef void (ompi_op_c_handler_fn_t)(const void *, void *, int *,
```

Comment:
Don't we need to change the signatures of the implementations that are used to set these in `ompi_op_t`?

Reply:
Which ones? I looked for implementations (`ompi/mca/op/base/op_base_functions.c` for example) and seems to me like all the functions already adhere to this prototype. Maybe I missed some?

Reply:
You're right, looks like the build is clean :+1: